js/filters/dateFilters.js

```diff
@@ -20,8 +20,42 @@
  *  
  */
 
+/* global app */
+/* global OC */
+/* global moment */
+
 app.filter('relativeDateFilter', function() {
 	return function (timestamp) {
 		return OC.Util.relativeModifiedDate(timestamp*1000);
 	}
 });
+
+app.filter('relativeDateFilterString', function() {
+	return function (date) {
+		return OC.Util.relativeModifiedDate(Date.parse(date));
+	}
+});
+
+app.filter('dateToTimestamp', function() {
+	return function (date) {
+		return Date.parse(date);
+	}
+});
+
+app.filter('parseDate', function() {
+	return function (date) {
+		if(moment(date).isValid()) {
+			return moment(date).format('YYYY-MM-DD');
```

Comment:
Yes, but that is out of scope for now, since we want to release 0.2 soon. We should also be timezone aware. But let's address that afterwords.